spec/factories/factories.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal :true
+FactoryBot.define do
+  factory :fixed_datum, :class => Models::FixedDatum do
+    year  { Faker::Number.between(2000, 2050) }
+    rc_uplift { Faker::Number.between(1, 2) }
+  end
+
+  factory :rent_charge, :class => Models::RentCharge do
+    uprn { Faker::Base.regexify(/[0-9]{3}[a-z]{3}/) }
+    address { Faker::Base.regexify(/[0-9]{1,2} [a-z]{5,10}/) }
+    address_2 { Faker::Base.regexify(/[A-Z]{1}[a-z]{5,10}/) }
+    comments { Faker::Base.regexify(/[a-z ]{15}/) }
+    rr_count { Faker::Number.between(1, 3) }
+    property_type { 'Flat' }
```

Comment:
Should probably generate random data for `property_type` too - I know I pointed you to the factory I created in the rent-charges-statistics branch which didn't have random property_type data, but I've since updated it if you want to have a look: https://github.com/madetech/rent-charges-api/blob/rent-charge-statistics/spec/factories/rent_charges.rb